Entity/Courses.py

```diff
@@ -22,7 +22,7 @@ class CollegeStanding(enum.Enum):
 class Courses(Base):
     __tablename__ = 'Courses'
     # TODO: update schema and this Courses class to follow snake_case convention
-    courseId = Column(Integer, primary_key=True)
+    courseId = Column(String(10), primary_key=True)
```

Comment:
Let's undo this line because the primary key should be an integer!

Reply:
good catch @Jason-Ku thank you!

Reply:
:thumbsup: